vault/raft.go

```diff
@@ -947,39 +947,28 @@ func (c *Core) JoinRaftCluster(ctx context.Context, leaderInfos []*raft.LeaderJo
 				}
 
 			case leaderInfo.AutoJoin != "":
+				// go-discover returns IPs
 				addrs, err := disco.Addrs(leaderInfo.AutoJoin, c.logger.StandardLogger(nil))
```

Comment:
Minor tweak, should I just rename `addrs` to `ips` to make this more obvious for future maintainers? Or just leave it as is to mimic disco's naming?

Reply:
ah good point, I agree that addrs is not clear on what the variable is and a rename might be nice, how does `clusterIPs` sound ?

Reply:
Sure thing!